src/features/content-answers/ContentAnswers.tsx

```diff
@@ -16,6 +16,7 @@ type Props = {
 };
 
 const ContentAnswers = ({ file }: Props) => {
+    const fileName = file && file.name;
```

Comment:
this should have been changed in master to pass `file`, please verify if you merged/rebase with master ocurred correctly